logging/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,44 @@
+# Troubleshooting
+
+## Issue: Fluent Bit runs without errors, but no logs are produced
+
+### Description
+If you move the Docker root directory in your Kubernetes cluster, symbolic links are created to the 
+directory from which Fluent Bit reads log files. However, Fluent Bit cannot read the files in the directory, 
+which are present only as links. Because Fluent Bit can see the directory, it does not issue an error 
+message, but it does not produce any logs to be displayed in Kibana.
```

Comment:
replace "produce" with "collect" or, maybe with "process".  They key point is that Fluent Bit is not _producing_ the log files in this location, it is reading the log messages and sending them on to Elasticsearch (to be surfaced in Kibana).

Reply:
Changed to "process". I didn't think "produce" was correct, but I wasn't sure what the correct action was.